apis/api/commontypes/astarte_common_types.go

```diff
@@ -344,6 +344,10 @@ type AstarteVerneMQSpec struct {
 	PersistentClientExpiration string `json:"persistentClientExpiration,omitempty"`
 	// +optional
 	MirrorQueue string `json:"mirrorQueue,omitempty"`
+	// This option allows to handle SSL termination at VerneMQ level.
+	// Default: false
+	// +optional
+	TerminateSSL *bool `json:"terminateSSL,omitempty"`
```

Comment:
I don't really like the semantics here, especially given you're adding a listener, de facto. I'd go for something like `SSLListener` or the likes.

Reply:
Replaced with `SSLListener`